src/main/scala/uk/gov/ons/bi/dataload/loader/SourceDataToParquetLoader.scala

```diff
@@ -27,28 +28,31 @@ class SourceDataToParquetLoader(ctxMgr: ContextMgr) {
     val appDataConfig = appConfig.AppDataConfig
     val workingDir = appDataConfig.workingDir
 
-    // Get directories and file names etc for specified data source
-    val (extSrcFile, extDataDir, parquetFile, tempTable) = biSource match {
-      case VAT => (extDataConfig.vat, extDataConfig.vatDir, appDataConfig.vat, "temp_vat")
-      case CH => (extDataConfig.ch, extDataConfig.chDir, appDataConfig.ch, "temp_ch")
-      case PAYE => (extDataConfig.paye, extDataConfig.payeDir, appDataConfig.paye, "temp_paye")
-    }
+     //Get directories and file names etc for specified data source
+        val (extSrcFile, extDataDir, parquetFile, tempTable) = biSource match {
+          case VAT => (extDataConfig.vat, extDataConfig.vatDir, appDataConfig.vat, "temp_vat")
+          case CH => (extDataConfig.ch, extDataConfig.chDir, appDataConfig.ch, "temp_ch")
+          case PAYE => (extDataConfig.paye, extDataConfig.payeDir, appDataConfig.paye, "temp_paye")
+        }
+
+    val inputPath = s"$extEnv/$extBaseDir/$extDataDir/$extSrcFile"
+    val outputPath = s"$workingDir/$parquetFile"
+    writeAdminParquet(inputPath, outputPath, tempTable, biSource)
```

Comment:
the method is called `load` but youre performing a write, put the write in another method `readWriteBusinessData`

Reply:
The write happens in writeAdminparquet, which is called by the load method.

Reply:
Whilst the actual write may happen in `writeAdminParquet` youre still calling it here which is misleading and the wrong place.